src/coreclr/nativeaot/Runtime/arm/GcProbe.asm

```diff
@@ -326,8 +286,8 @@ __PPF_ThreadReg SETS "r2"
     NESTED_ENTRY RhpGcProbeRare
```

Comment:
Delete `RhpGcProbe` and rename `RhpGcProbeRare` to `RhpGcProbe` to match other platforms?

Reply:
I did not put much effort in keeping 32bit platforms in sync. I only did some easy opportunistic changes.

I think we may be iterating on 64bit implementation a bit more and keeping 32bit behind is not a big deal. When/if we get to 32bit, absorbing all the changes should not be hard. It might even be that porting 64bit code, that works and tested, would be a shorter path than adjusting existing 32bit code. Probably it will be a mix of both approaches.
 
I will check if these changes are easy to do.

Reply:
I have fixed the entry points so that all platforms look the same - one `RhpGcProbeHijack` that checks for the trap flag, and if set, sets bits for saved registers and calls `RhpWaitForGC`. It might even work when we get to 32bit.